src/routes/files/+page.svelte

```diff
@@ -542,14 +589,38 @@
                                         deleteItem(item.name)
                                     },
                                 },
+                                {
+                                    id: "rename-" + item.id,
```

Comment:
Can we replace this with `rename-${item.id}` just so we use the same concatenation everywhere